### PR TITLE
opslevelsignature: add go example

### PR DESCRIPTION
**Pull request by nvanheuverzwijn**
This add a golang example on how to verify opslevel signature

### CODE REVIEW

File: scripts/verify_opslevel_webhook_signature/go/go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module opslevelsignature
+
+go 1.20
```
Comment (davidbloss) [inline scripts/verify_opslevel_webhook_signature/go/go.mod:3]:
```suggestion
go 1.23
```
Bumping go version to latest and tested that it still works. `1.20` is EOL'd [link](https://endoflife.date/go)

File: scripts/verify_opslevel_webhook_signature/go/signature.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+)
+
+const (
+	HeaderSignatureCanonical = "X-Opslevel-Signature"
+	HeaderSignature          = "X-OpsLevel-Signature"
+	HeaderTimingCanonical    = "X-Opslevel-Timing"
+	HeaderTiming             = "X-OpsLevel-Timing"
+)
+
+// Search for opslevle signature
```
Comment (jasonopslevel) [inline scripts/verify_opslevel_webhook_signature/go/signature.go:20]:
```suggestion
// Search for OpsLevel signature
```

Small spelling correction, with some Capitalization thrown in for good measure :)

**Review by jasonopslevel — DISMISSED** [review]
LGTM! Tested the full flow locally and works great. 

Thanks a lot for your help!

**Review by davidbloss — APPROVED** [review]
LGTM 🎉

**Pull request merged**